clap_complete_fig/src/fig.rs

```diff
@@ -216,8 +216,7 @@ fn gen_options(cmd: &Command, indent: usize) -> String {
                 buffer.push_str(&format!("{:indent$}],\n", "", indent = indent + 4));
             }
 
-            #[allow(deprecated)]
-            if option.is_multiple_occurrences_set() {
```

Comment:
I leave this to you all but for the clap 3 time frame, I've been leaving in the deprecated feature support for completions.  Not everyone is using the new actions and checking `option.is_multiple_occurrences_set` is important for those applications.

Reply:
Also, once clap 4 is released, everything will be multiple occurrences.  I would instead frame this in terms of `if !old action or multiple occurrences set`.  That way when the deprecated code is removed, we remove the `if` wrapping the body and the logic is enabled by default.

Reply:
Will fix! Thanks @epage!

Reply:
Not 100% sure this is what you mean, but I added an explicit check for the old action types which can be remove once clap 4 is release.

Reply:
lose enough